scripts/anonymize.py

```diff
@@ -104,7 +107,7 @@ def sysprint(command):
 
 
 def main():
-    now = datetime.now().strftime('%Y%m%d')
+    now = timezone.now().strftime('%Y%m%d')
```

Comment:
this is a script and doesn't run with a full Django settings environment, so no access to timezone. also doesn't make sense to use it since strftime will just get rid of the timezone info in this case.